### ISSUE TITLE
setquot3

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#1000]
Here is another elegant way to construct set quotients: http://www.cs.bham.ac.uk/~mhe/agda-new/UF-Quotient.html , devised by Martin Escardo.  We should implement it, perhaps after waiting for us to stop using type-in-type, so we can check the universe levels, too.

**Comment by martinescardo** [comment]
This is in github as well, here https://github.com/martinescardo/TypeTopology

I did discover this method myself, but it is in the HoTT Book too, independently. What the file @DanGrayson links tries to do is to explain (to Dan) how the universe levels work in Agda when we perform this construction.

**Comment by arnoudvanderleer** [comment]
@DanGrayson Is this still a thing that we want (why?) And if so, can you summarize the approach?
